amundsen_application/static/js/ducks/search/api/v0.ts

```diff
@@ -42,16 +42,15 @@ export function searchResource(
 ) {
   /* If resource support is not configured or if there is no search term for non-filter supported resources */
   if (
-    (resource === ResourceType.dashboard &&
-      (!indexDashboardsEnabled() || term.length === 0)) ||
+    (resource === ResourceType.dashboard && !indexDashboardsEnabled()) ||
     (resource === ResourceType.user &&
       (!indexUsersEnabled() || term.length === 0))
   ) {
     return Promise.resolve({});
   }
 
   /* Note: This logic must exist until query string endpoints are created for all resources */
-  if (resource === ResourceType.table) {
+  if (resource === ResourceType.table || resource === ResourceType.dashboard) {
```

Comment:
101qq: is this the main line to handle search filter for both dashboard and table? Per my read, we don't have any special logic to handle table filter vs dashboard filter in JS code?

Reply:
Not particularly in the JS code. The filter configuration sets up the specific filter categories, and the application keeps track of the state of each one based on what the user has entered. This logic cares about what kind of filter it is (e.g. single input or multi-select checkbox) but  not the actual content. 

Then on line 55 that information will either get sent to `/api/search/v0/table` or `/api/search/v0/dashboard` in the Flask layer.